src/main/java/com/school/project/model/entities/Module.java

```diff
@@ -1,13 +1,35 @@
 package com.school.project.model.entities;
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
 import java.util.List;
 
+@Entity
+@Table(name = "MODULE")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+
 public class Module {
 
+    @Id
+    @Column(name = "ID")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+    @Column(name = "NAME", nullable = false)
     private String name;
+    @Column(name = "HOURS", nullable = false)
     private Integer hours;
+    @OneToMany(cascade = CascadeType.ALL, targetEntity = Subject.class)
```

Comment:
hypothetical we may have same subjects in different modules, so I think we have Many to Mane relation have